docs/configuration.md

```diff
@@ -753,7 +753,7 @@ interpretation of the options.
       no
     </td>
     <td>
-      Duration for which a signed URL should be valid.
+      Specify a `duration` by provding an integer and a unit. Valid time units are `ns`, `us` (or `µs`), `ms`, `s`, `m`, `h`. For example, `3m` is a valid duration. If you omit this value or do not specify a time unit, this defaults to 20 minutes.
```

Comment:
I suggest replacing:

> If you omit this value or do not specify a time unit, this defaults to 20 minutes.

to 

>Defaults to 20 minutes if the whole `duration` configuration setting is omitted

In other words, I suggest we don't provide the possibility for users to not specify a time unit or value at all, even if the code currently behaves this way and defaults to 20mins for unitless values. The reasoning is to urge users to use the config file parameters correctly with the right unit and avoid reliance on a behavior in the code that might change in the future [1]

[1]: as mentioned earlier currently if `duration` is defined as a string [it fails to get parsed](https://github.com/docker/distribution/blob/master/registry/storage/driver/middleware/cloudfront/middleware.go#L96) and the default value, `20*time.Minute` remains.